apps/web/src/app/[lang]/app/[type]/parties/[partyName]/[partyId]/individuals-table/form.tsx

```
@@ -0,0 +1,140 @@
+import type { AutoFormInputComponentProps } from "@repo/ayasofyazilim-ui/organisms/auto-form";
+import AutoForm, {
+  AutoFormSubmit,
+  CustomCombobox,
+} from "@repo/ayasofyazilim-ui/organisms/auto-form";
+import { useRouter } from "next/navigation";
+import { useEffect, useState } from "react";
+import {
+  getAffiliationCodeApi,
+  getIndividualsApi,
+} from "src/app/[lang]/app/actions/CrmService/actions";
+import { postAffiliationsApi } from "src/app/[lang]/app/actions/CrmService/post-actions";
+import type { AffiliationsPostDto } from "src/app/[lang]/app/actions/CrmService/types";
+import { handlePostResponse } from "src/app/[lang]/app/actions/api-utils-client";
+import type { CRMServiceServiceResource } from "src/language-data/CRMService";
+import type { PartyNameType } from "../../../types";
+
+interface AutoFormValues {
+  email: UniRefund_CRMService_EmailCommonDatas_UpdateEmailCommonDataDto;
+  affilation: AffiliationsPostDto;
+}
+function AffiliationsForm({
+  languageData,
+  partyName,
+  partyId,
+}: {
+  languageData: CRMServiceServiceResource;
+  partyName: Exclude<PartyNameType, "individuals">;
+  partyId: string;
+}) {
+  const [affiliationCodes, setAffiliationCodes] = useState<
+    UniRefund_CRMService_AffiliationCodes_AffiliationCodeDto[]
+  >([]);
+  const router = useRouter();
+
+  const affiliationsSchema = createZodObject(
+    {
+      type: "object",
+      properties: {
+        email: $UniRefund_CRMService_EmailCommonDatas_UpdateEmailCommonDataDto,
+        affilation:
+          $UniRefund_CRMService_AffiliationCodes_CreateAffiliationCodeDto,
+      },
+    },
+    undefined,
+    undefined,
+    {
+      email: ["emailAddress"],
+      affilation: ["roleId"],
+    },
+  );
+
+  useEffect(() => {
+    void getAffiliationCodeApi("individuals").then((response) => {
+      if (response.type === "success") {
+        setAffiliationCodes(response.data.items || []);
+        return;
+      }
+      toast.error(languageData["Fetch.Fail"]);
```

can we return the error type from the backend?